### ISSUE TITLE
groups: migration in inconsistent state

### ISSUE THREAD

**Issue by jalehman** [tloncorp/landscape-apps#1402]
The internal UF group is hosted on `~wolref-podlex`. From within `~wolref-podlex` all channels hosted on wolref appear to have successfully migrated to the new groups.

<img width="263" alt="Screenshot 2022-12-09 at 10 57 55 AM" src="https://user-images.githubusercontent.com/550978/206775462-b1d7549a-0028-497f-b53a-acda9c49d498.png">

`~rovnys-ricfer`, who is also in that group, sees all of those channels as pending migration:

![image](https://user-images.githubusercontent.com/550978/206774786-47f1e71f-c1e0-4d93-8447-9c9ad1154799.jpg)

The exception is with two channels that are not hosted on `~wolref-podlex` that `~rovnys-ricfer` has successfully joined. From `~wolref-podlex`, I'm unable to even see either of those channels.

**Comment by jalehman** [comment]
Update: turns out that no one on our team is able to use any of the channels (post, receive messages) in this group. This group is our primary means of communication which makes this a pretty major issue. I know it's got top severity already, but consider me available for whatever is needed on our end to help resolve this.

**Comment by arthyn** [comment]
@jalehman can you confirm you've ran this? https://groups.google.com/a/urbit.org/g/dev/c/RTrZ2tCMF64

**Comment by belisarius222** [comment]
For what it's worth, my landscape is at `6to5e`, which I think is the latest.

**Comment by jalehman** [comment]
Huh! I've got something completely different:

```
> +vat %landscape
%landscape
  /sys/kelvin:      [%zuse 417]
  base hash:        0v7.e9gor.edog8.u6aet.12o5d.1kngj.fdcnp.1513i.bv1ce.qlrtn.0bti0
  %cz hash:         0v7.e9gor.edog8.u6aet.12o5d.1kngj.fdcnp.1513i.bv1ce.qlrtn.0bti0
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~lander-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~doznec-dozzod-marzod
  source desk:      %landscape
  source aeon:      7
  kids desk:        ~
  pending updates:  ~
::
```

**Comment by jalehman** [comment]
Nvm, that appears to be this separate issue: https://github.com/tloncorp/landscape-apps/issues/1442

**Comment by arthyn** [comment]
@belisarius222 @thomasurbit can y'all run `:group-store +dbug [%state '(~(has in wait) ~wolref-podlex)']`?

**Comment by belisarius222** [comment]
```
%group-store: on-peek on path /x/dbug/state
%group-store: on-save
>   %.y
> :group-store +dbug [%state '(~(has in wait) ~wolref-podlex)']
>=
~rovnys-ricfer:dojo> 
```

**Comment by belisarius222** [comment]
Looks like my %landscape updated itself sometime before I ran that last command:
```
> +vat %landscape
%landscape
  /sys/kelvin:      [%zuse 417]
  base hash:        0vp.llhuf.pg57r.5piau.ea5hr.pq5ct.b79c0.3q6hl.g0n6o.n5jlu.jq6nu
  %cz hash:         0vp.llhuf.pg57r.5piau.ea5hr.pq5ct.b79c0.3q6hl.g0n6o.n5jlu.jq6nu
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~lander-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~lander-dister-dozzod-dozzod
  source desk:      %landscape
  source aeon:      33
  kids desk:        ~
  pending updates:  ~
::
```

**Comment by belisarius222** [comment]
Now whenever I open the group in the UI, I see these in the dojo, one per unjoinable channel in the group:
```
%diary: on-peek on path /x/diary/~wolref-podlex/operations-1477/perm
peek bad result
/app/diary/hoon:<[554 27].[554 46]>
/app/diary/hoon:<[554 5].[554 47]>
/app/diary/hoon:<[513 14].[513 41]>
/app/diary/hoon:<[513 6].[513 41]>
/app/diary/hoon:<[513 5].[513 55]>
/app/diary/hoon:<[512 5].[513 55]>
/app/diary/hoon:<[511 5].[513 55]>
/app/diary/hoon:<[504 3].[523 5]>
/app/diary/hoon:<[503 3].[523 5]>
/sys/vane/gall/hoon:<[1.136 26].[1.136 60]>
%chat: on-peek on path /x/chat/~wolref-podlex/watercooler-2457/perm
peek bad result
/app/chat/hoon:<[958 26].[958 45]>
/app/chat/hoon:<[958 5].[958 46]>
/app/chat/hoon:<[633 14].[633 41]>
/app/chat/hoon:<[633 6].[633 41]>
/app/chat/hoon:<[633 5].[633 55]>
/app/chat/hoon:<[632 5].[633 55]>
/app/chat/hoon:<[631 5].[633 55]>
/app/chat/hoon:<[621 3].[684 5]>
/app/chat/hoon:<[620 3].[684 5]>
/sys/vane/gall/hoon:<[1.136 26].[1.136 60]>
```

**Comment by arthyn** [comment]
that output makes sense because you don't actually have those channels yet, because you're still waiting on josh

**Comment by tomholford** [comment]
Now also seeing the `on-peek` spam, not sure when this started. It's for a couple groups that I host, both public and private. Is there any diagnostic info I can collect that would be helpful?

```
...
%group-store: on-peek on path /x/groups/ship/~datder-sonnet/
...
```

**Comment by arthyn** [comment]
@tomholford how often? it's very loud when someone updates a contact because all hosts start pinging everyone they know to update. if it's not a constant stream it's probably normal.

**Comment by tomholford** [comment]
> @tomholford how often? it's very loud when someone updates a contact because all hosts start pinging everyone they know to update. if it's not a constant stream it's probably normal.

Non-stop, makes it tough to enter dojo commands

**Comment by jalehman** [comment]
@jamesacklin this doesn't seem completed to me, as the one with the issue

**Comment by arthyn** [comment]
@jalehman yeah I think this one was mistakenly completed.
